Libplanet/Store/IStore.cs

```diff
@@ -16,6 +16,8 @@ public interface IStore
 
         long CountIndex(string @namespace);
 
+        long RecentIndex(string @namespace);
```

Comment:
This method doesn't seem used anywhere.  Is this necessary?

Reply:
Well, of course not. It was a mistake.
I will going to delete the code associated with the method.